lessons/pydata/ml_intro/index.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<p>Bylo by možná dobré říct něco o tom, co od našeho modelu očekáváme / můžeme očekávát, např. vzhledem k nemalému rozptylu a jen jedné vysvětlující proměnné. A co vlastně vzít všechny proměnné? V klasifikaci jsme to už zvládli :) Grafy budou sice složitější, ale dá se třeba vybrat pár proměnných (aspoň bude vidět že závislost není jednoduše jen na jedné) a výsledek se dá ukázat v jednom grafu predikce vs skutečnost.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
více proměnných vzít můžu, snažila jsem se to co nejvíc zjednodušit - má to být jen ukázka toho, co to je klasifikace a co to je regrese, více do hloubky se chci pouštět až na těch reálných datech. Ale promyslím to.